src/mod_pushoff.erl

```diff
@@ -392,15 +406,29 @@ parse_backends(Plists) ->
 parse_backend(Opts) ->
     RawType = proplists:get_value(type, Opts),
     Type =
-        case lists:member(RawType, [apns]) of
+        case lists:member(RawType, [apns, fcm]) of
             true -> RawType
         end,
-    CertFile = proplists:get_value(certfile, Opts),
     Gateway = proplists:get_value(gateway, Opts),
+    CertFile = 
```

Comment:
consider:

```erlang
CertFile = proplists:get_value(gateway, Opts),
ApiKey = proplists:get_value(api_key, Opts),
```

as:

```
1> proplists:get_value(certfile, [{key, value}]).
undefined
```

it's also best to make have it as an atom and cause runtime type errors instead of having them as empty strings and cause other kinds of bizarre errors.